lmnet/executor/export.py

```diff
@@ -43,7 +43,7 @@ def _pre_process(raw_image, pre_processor, data_format):
     return image
 
 
-def _save_npy(image_path, output_dir, image, raw_image, all_outputs, image_size):
+def _save_inference_values_foreach_operation_as_npy(image_path, output_dir, image, raw_image, all_outputs, image_size):
```

Comment:
I think the name may be too long, makes it hard to read.

Reply:
@nlpng 
How is `_save_all_operation_outputs()` ?

Reply:
Thanks, this sounds more recognizable.